status.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Git keeps track of whether a file is executable. You should be able to chmod +x it, then add and commit it.